### PR TITLE
Update public_suffix_list.dat

### PR DESCRIPTION
**Pull request by ddnss**
Add DynDNS -Service Domains to public suffix List 

DynDNS Service is an dynamic DNS services under the Domains ddnss.org
ddnss.de
dyn.ddnss.de
dyndns.ddnss.de
dyndns1.de
dyn-ip24.de
home-webserver.de
dyn.home-webserver.de
myhome-server.de

For security reasons (cookie isolation) and to provide them with the ability to create Let's Encrypt certificates, we would like to add those domains to the public suffix list.

### CODE REVIEW

**Comment by ddnss** [comment]
dig +short psl.ddnss.de TXT
"https://github.com/publicsuffix/list/pull/180"

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11052,6 +11052,20 @@ webhop.org
 worse-than.tv
 writesthisblog.com
 
+// ddnss.de : https://www.ddnss.de/
+// Submitted by Robert Niedziela <webmaster@ddnss.de> 2016-03-14
+
```
Comment (sleevi) [inline public_suffix_list.dat]: Please remove this blank line.

File: public_suffix_list.dat [hunk 2]
```diff
@@ -11052,6 +11052,20 @@ webhop.org
...
+ddnss.de
+ddnss.org
+dyn.ddnss.de  
+dyndns.ddnss.de  
+dyndns1.de
+dyn-ip24.de
+home-webserver.de  
+myhome-server.de  
+dyn.home-webserver.de
+
```
Comment (sleevi) [inline public_suffix_list.dat]: Please remove this blank line.

File: public_suffix_list.dat [hunk 3]
```diff
@@ -11052,6 +11052,20 @@ webhop.org
...
+ddnss.de
+ddnss.org
+dyn.ddnss.de  
+dyndns.ddnss.de  
+dyndns1.de
+dyn-ip24.de
+home-webserver.de  
+myhome-server.de  
+dyn.home-webserver.de
```
Comment (sleevi) [inline public_suffix_list.dat]: Please keep the suffices in alphabetical order.

In this case,
home-webserver.de
dyn.home-webserver.de
myhome-server.de

(Sorting from the right-most, most generic label, to the left-most, most specific label)

**Comment by sleevi** [comment]
Can you please provide a brief description of why you want these suffix(es) added to the list, as explained on https://github.com/publicsuffix/list/wiki/Guidelines

File: public_suffix_list.dat (hunk 3 above)
Comment (ddnss) [inline public_suffix_list.dat]: A,B C D E F GH????

**Comment by 3mb3dw0rk5** [comment]
@ddnss Any progress at this?

**Comment by ddnss** [comment]
Updatet

**Comment by mbergner** [comment]
To tentatively answer the open question, ddnss.de (and the other domains) hosts a dynamic DNS service.

**Comment by mbergner** [comment]
@ddnss Additionally, according to the [guidelines](https://github.com/publicsuffix/list/wiki/Guidelines), you need to add a TXT record to all of the domains specified in the pull request, which I don't see: 

    dig +short _psl.ddnss.de TXT
    dig +short _psl.dyn-ip24.de TXT

and all the other domains return nothing.

**Comment by weppos** [comment]
Unfortunately we can't merge this PR as it doesn't follow the guidelines
https://github.com/publicsuffix/list/wiki/Guidelines
and because I was unable to validate the DNS records as requested by the Guidelines as well.

**Comment by ddnss** [comment]
ok, should be done now!
thx

dig +short _psl.dyn-ip24.de TXT
"https://github.com/publicsuffix/list/pull/180"

dig +short _psl.ddnss.de TXT
"https://github.com/publicsuffix/list/pull/180"

+short _psl.ddnss.ch TXT
"https://github.com/publicsuffix/list/pull/180"

 dig +short _psl.dyndns1.de TXT
"https://github.com/publicsuffix/list/pull/180"

dig +short _psl.ddnss.org TXT
"https://github.com/publicsuffix/list/pull/180"

dig +short _psl.dyn.ddnss.de TXT
"https://github.com/publicsuffix/list/pull/180"
,
dig +short _psl.dyndns.ddnss.de TXT
"https://github.com/publicsuffix/list/pull/180"

dig +short _psl.home-webserver.de TXT
"https://github.com/publicsuffix/list/pull/180"

 dig +short _psl.dyn.home-webserver.de TXT
"https://github.com/publicsuffix/list/pull/180"

dig +short _psl.myhome-server.de TXT
"https://github.com/publicsuffix/list/pull/180"

**Comment by sleevi** [comment]
While I can verify the domain, this PR still does not adhere to the [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines). 

In particular, there are errors with sorting and with insufficient whitespace between other entries. Please update the PR.

**Comment by ddnss** [comment]
Ok thanks, iam pull new request

Von meinem iPhone gesendet

File: public_suffix_list.dat [hunk 4]
```diff
@@ -11052,6 +11052,17 @@ webhop.org
 worse-than.tv
 writesthisblog.com
 
+// ddnss.de . : https://www.ddnss.de/
```
Comment (sleevi) [inline public_suffix_list.dat]: Unnecessary `.` here

Also, please list the corporate identity, if any

File: public_suffix_list.dat [hunk 5]
```diff
@@ -11052,6 +11052,17 @@ webhop.org
...
+// Submitted by Robert Niedziela <webmaster@ddnss.de>
+ddnss.de
+dyn.ddnss.de
+dyndns.ddnss.de
+dyndns1.de
+dyn-ip24.de
+home-webserver.de
+dyn.home-webserver.de
+myhome-server.de
+ddnss.org
```
Comment (sleevi) [inline public_suffix_list.dat]: There should be a newline between the last domain here for yours and the start of the next section.

File: public_suffix_list.dat [hunk 6]
```diff
@@ -11052,8 +11052,8 @@ webhop.org
 worse-than.tv
 writesthisblog.com
 
-// ddnss.de . : https://www.ddnss.de/
-// Submitted by Robert Niedziela <webmaster@ddnss.de>
+// ddnss.de : https://www.ddnss.de/
+// Submitted by Robert Niedziela <webmaster@ddnss.de> 2016.11.07
```
Comment (sleevi) [inline public_suffix_list.dat]: Ahh! You were doing good until you added this! :) Remove the date :)

**Comment by ddnss** [comment]
it´s merged ?

+ ERROR: An error occurred while sending post-request to https://acme-v01.api.letsencrypt.org/acme/new-cert (Status 429)

Details:
{
  "type": "urn:acme:error:rateLimited",
  "detail": "Error creating new cert :: Too many certificates already issued for: Domain.de",
  "status": 429
}

**Comment by gerv** [comment]
You will have to ask Let's Encrypt how often they update their copy of the list.

Again, the PSL is not a "Let's Encrypt Permissions Service". They have a form for that - use it.

**Comment by ddnss** [comment]
Thanks :)

-----Ursprüngliche Nachricht-----
Von: Gervase Markham [mailto:notifications@github.com] 
Gesendet: Donnerstag, 17. November 2016 16:28
An: publicsuffix/list
Cc: ddnss; Mention
Betreff: Re: [publicsuffix/list] Update public_suffix_list.dat (#180)

You will have to ask Let's Encrypt how often they update their copy of the list.

Again, the PSL is not a "Let's Encrypt Permissions Service". They have a form for that - use it.

**Comment by ddnss** [comment]
hi, 

is here an Workarround about get Cers from others than Let´s encrypt for root Domain ?
because there now Domains unaviable for others if you on public Suffix

or the hardway remove from list?

**Pull request merged**